docs/src/API_GRPC_Explanation.md

```
@@ -88,6 +91,44 @@ Please note that fees will be applicable for every bit used in the `payment_id`.
 #### Encoding
 After serialization, the complete byte array is encoded using **Base58**, resulting in a human-readable Tari address.
 
+Please note that Tari supports three address formats for representation of the address:
+- Hexidecimal
+- Base58
+- Emoji ID
+
+**EmojiID**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)